research/query_service/ir/graph_proxy/src/apis/graph/mod.rs

```diff
@@ -40,6 +41,27 @@ pub fn write_id<W: WriteExt>(writer: &mut W, id: ID) -> io::Result<()> {
 /// The number of bits in an `ID`
 pub const ID_BITS: usize = std::mem::size_of::<ID>() * 8;
 
+/// Primary key in storage, including single column pk and multi column pks.
+#[derive(Debug)]
+pub enum PK {
```

Comment:
`PKV `? They are both key and values.  BTW, there is an `OneOrMany<T>` in meta.rs. Consider move it to ir_common, and then define `PKV` as `OneOrMany<(NameOrId, Object)>`. There are good things about `OneOrMany<T>`, it can be `as_ref()` to `&[T]`.

Reply:
Done